stdlib/public/Python/Python.swift

```diff
@@ -1257,6 +1257,30 @@ extension PythonObject : MutableCollection {
   }
 }
 
+extension PythonObject : Sequence {
+  public struct Iterator : IteratorProtocol {
+    fileprivate let pythonIterator: PythonObject
+
+    public func next() -> PythonObject? {
+      guard let result = PyIter_Next(self.pythonIterator.borrowedPyObject) else {
+        try! throwPythonErrorIfPresent()
+        return nil
+      }
+
+      defer {
+        Py_DecRef(result)
```

Comment:
This could cause a segfault. `PythonObject.init(consuming:)` already takes ownership of a `PyObject` pointer and calls `Py_DecRef` in its `deinit`. Remove this defer block and then you are good.

Also, there's no need to add empty lines in this compact function.